lib/models/difficulty.dart

```diff
@@ -0,0 +1,5 @@
+enum Difficulty {
+  easy,
```

Comment:
Capitalize the values of the enum

Reply:
Referring to this [link](https://dart.dev/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names)

> In new code, use lowerCamelCase for constant variables, including enum values.

Shouldn't we follow this convention?

Reply:
I forgot about this convention. I use java a lot and the java convention is to use capital letters for all static or enum variables such as EASY.

Since this is a flutter project, let's follow lowerCamelCase. So no changes are required in the 'difficulty.dart' file.